src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+function createPromise() {
+  const result = document.querySelector('body');
```

Comment:
you can move it outside a function to have it in main scope to be visible for other functions etc
and to not duplicate it